### ISSUE TITLE
Coherent super resolved radar beamforming using self-supervised learning. (arXiv:2106.13085v1 [eess.SP])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#8237]
Coherent, super resolved radar beamforming using self-supervised learning. (arXiv:2106.13085v1 [eess.SP])<br>
https://ift.tt/2Ssta04<br>
<p>High resolution automotive radar sensors are required in order to meet the high bar of autonomous vehicles needs and regulations. However, current radar systems are limited in their angular resolution causing a technological gap. An industry and academic trend to improve angular resolution by increasing the number of physical channels, also increases system complexity, requires sensitive calibration processes, lowers robustness to hardware malfunctions and drives higher costs. We offer an alternative approach, named Radar signal Reconstruction using Self Supervision (R2-S2), which significantly improves the angular resolution of a given radar array without increasing the number of physical channels. R2-S2 is a family of algorithms which use a Deep Neural Network (DNN) with complex range-Doppler radar data as input and trained in a self-supervised method using a loss function which operates in multiple data representation spaces. Improvement of 4x in angular resolution was demonstrated using a real-world dataset collected in urban and highway environments during clear and rainy weather conditions.</p>
<br>
<br>
via cs.RO updates on arXiv.org http://arxiv.org/<br>
